### PR TITLE
Add RoadMap for 2020

### PR DESCRIPTION
**Pull request by waveywaves**
# Changes

Based on the May 21st Community Call, this Roadmap has been added.
This PR would have to be open for a week or two atleast to gain enough insight and clarity to understand if these are the things which we would like to aim for wrt the future of the Operator.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes tests (if functionality changed/added)
- [x] Includes docs (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/jenkinsci/kubernetes-operator/blob/master/CONTRIBUTING.md#commit-messages)

_See [the contribution guide](https://github.com/jenkinsci/kubernetes-operator/blob/master/CONTRIBUTING.md) for more details._

## Reviewer Notes

If API changes are included, additive changes must be approved by at least two [OWNERS](https://github.com/jenkinsci/kubernetes-operator/blob/master/OWNERS) and backwards incompatible changes must be approved by [more than 50% of the OWNERS](https://github.com/jenkinsci/kubernetes-operator/blob/master/OWNERS).

# Release Notes

```
Describe any user facing changes here, or delete this block.

Examples of user facing changes:
- API changes
- Bug fixes
- Any changes in behavior

```

### CODE REVIEW

**Comment by waveywaves** [comment]
This PR needs to be open for a week or two atleast to have enough discussions, insights and updates to the aforementioned topics to gain enough clarity and vision.

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Jenkins Operator Roadmap 2020
+
+## Goals
+
+- Move from Pod to Deployment based Jenkins Instance creation
+- Reconciliation on the Jenkins CR updates the Status and not the Spec
+- Jenkins Image Controller
```
Comment (oleg-nenashev) [inline ROADMAP.md:7]: Would be great to expand it a bit

File: ROADMAP.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+- Jenkins Operator Openshift Support 
+- Airgapped Environment Support
```
Comment (oleg-nenashev) [inline ROADMAP.md:9]: Does it mean setting up the internal update center?

**Comment by waveywaves** [comment]
I think this is ready for a final review.

**Comment by waveywaves** [comment]
Hi @tomaszsek can you check this out as well ?

**Comment by tomaszsek** [comment]
A new road map has been added recently, closing.

**Pull request closed without merging**